### PR TITLE
Update instructions for adding a feature flag for SettingsManifest

### PR DESCRIPTION
**Pull request by bion**

### CODE REVIEW

File: docs/contributor-guide/developer-guide/development-standards.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ We anticipate relatively low [QPS](https://en.wikipedia.org/wiki/Queries\_per\_s
 
 Exception handling in asynchronous execution deserves a special mention as `CompletionException` is unchecked and can be easily missed. We should always explicitly catch `CompletionException` when joining a future in a synchronous context. When returning `CompletionStage<Result>` to users, we should provide exception handling through `CompletableFuture#exceptionally` API.
 
+### Server config values
+
+Server config values are specifid in [server/conf/env-var-docs.json](https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/conf/env-var-docs.json) and read by the server using [Play's HOCON config system](https://playframework.com/documentation/2.8.x/Configuration). When new config variables are added, we update a generated [manifest file](https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/app/services/settings/SettingsManifest.java) that provides accessor methods. Developers should consume all config values through this class and avoid working directly with the `import com.typesafe.config.Config` class in application code.
```
Comment (rockycodes) [inline docs/contributor-guide/developer-guide/development-standards.md:64]:
```suggestion
Server config values are specified in [server/conf/env-var-docs.json](https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/conf/env-var-docs.json) and read by the server using [Play's HOCON config system](https://playframework.com/documentation/2.8.x/Configuration). When new config variables are added, we update a generated [manifest file](https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/app/services/settings/SettingsManifest.java) that provides accessor methods. Developers should consume all config values through this class and avoid working directly with the `import com.typesafe.config.Config` class in application code. 
```

File: docs/contributor-guide/developer-guide/feature-flags.md [hunk 2]
```diff
@@ -11,6 +11,11 @@ features such as
 Currently we only have boolean flags, but others could be added as
 necessary.
 
+The generated
+[`SettingsManifest`](https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/app/services/settings/SettingsManifest.java)
+class provides readable accessors for environment variables and when a `Request`
```
Comment (rockycodes) [inline docs/contributor-guide/developer-guide/feature-flags.md:16]:
```suggestion
class provides readable accessors for environment variables. When a `Request`
```

File: docs/contributor-guide/developer-guide/feature-flags.md [hunk 3]
```diff
@@ -34,6 +39,15 @@ The following process should be used to implement a feature flag, release it in
 11. Leave the flag in the codebase for at least another month after it went live in production (e.g. Seattle).
 12. Remove the flag, remove the guards in the code looking for the flag, note in the PR release notes that the flag is being removed, then close the Github issue. Ensure the email sent for the next release includes a note about the removal of the feature flag.
 
+
+### Adding a new flag
+
+1.  Add the flag to the `"Feature Flags"` section of [server/conf/env-var-docs.json](https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/conf/env-var-docs.json). 
+1.  Run `bin/fmt` to regenerated `SettingsManifest.java`
+1.  Inject `SettingsManifest` where you want to consume the flag
+1.  Call the generated method on `SettingsManifest` that matches the name of your flag. Unless infeasible, use the version of the method that takes a `play.mvc.Http.Request` argument so that the flag can be overridden in non-prod environments.
```
Comment (rockycodes) [inline docs/contributor-guide/developer-guide/feature-flags.md:48]: suggestion (non-blocking): give an example of a flag + how to invoke the getter method for it

**Review by rockycodes — APPROVED** [review]
looks great! sooo excited to take out those notes about passing it through the deploy system.

File: docs/contributor-guide/developer-guide/feature-flags.md (hunk 3 above)
Comment (bion) [reply docs/contributor-guide/developer-guide/feature-flags.md]: Done

**Pull request merged**